leptonai/photon/photon.py

```diff
@@ -892,12 +906,18 @@ def _mount_route(self, app, path, func):
             has_flask = False
 
         if isinstance(subapp, FastAPI):
-            app.include_router(subapp.router, prefix=f"/{path}")
+            if force_asgi_proxy:
```

Comment:
I think this is unrelated to asgi_proxy, maybe call it `mount_method="mount"` or `mount_method="include_router"`?

Reply:
My fault, haven't completely updated the diff yet... let me update

Reply:
Changed this to "use_router_if_possible=True", since this only applies to when the subapp is a fastapi app.